content/en/docs/getting-started/_index.md

```diff
@@ -60,73 +60,162 @@ olm-operator       1/1     1            1           5m52s
 packageserver      2/2     2            2           5m43s
 ```
 
-## Installing an Operator using OLM
+## Installing OLM with sdk
```

Comment:
Capitalize SDK like we do elsewhere in the doc since it's an acronym. Or better yet use "Operator SDK".
```suggestion
## Installing OLM with Operator SDK
```